From 587b01428d6ec2be1ab6b468198663118d483990 Mon Sep 17 00:00:00 2001
From: Hiltjo Posthuma <hiltjo@codemadness.org>
Date: Fri, 1 Mar 2019 11:49:29 +0100
Subject: [PATCH 5/8] fix issues in example scripts

- in case cd fails don't continue (don't write in previous directory).
- post-receive: quote $(pwd) in case it has spaces.

found by shellcheck tool.
---
 example_create.sh       | 2 +-
 example_post-receive.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/example_create.sh b/example_create.sh
index aff7d54..cf16e1d 100755
--- a/example_create.sh
+++ b/example_create.sh
@@ -30,7 +30,7 @@ for dir in "${reposdir}/"*/; do
 	printf "%s... " "${d}"
 
 	mkdir -p "${curdir}/${d}"
-	cd "${curdir}/${d}"
+	cd "${curdir}/${d}" || continue
 	stagit -c ".cache" "${reposdir}/${r}"
 
 	# symlinks
diff --git a/example_post-receive.sh b/example_post-receive.sh
index 0655569..c9bcb5e 100755
--- a/example_post-receive.sh
+++ b/example_post-receive.sh
@@ -14,7 +14,7 @@ export LC_CTYPE="en_US.UTF-8"
 
 name="$1"
 if test "${name}" = ""; then
-	name=$(basename $(pwd))
+	name=$(basename "$(pwd)")
 fi
 
 # config
-- 
2.21.0

